package.json

```diff
@@ -24,7 +24,13 @@
     "prepare-release": "./prepare-release.sh",
     "release": "./release.sh",
     "prune-arc-build": "node ./pruneArcBuild.js",
-    "postinstall": "npm run prune-arc-build"
+    "postinstall": "npm run prune-arc-build",
+    "verify.initialize": "npm explore @daostack/arc -- npm install openzeppelin-solidity @daostack/infra --no-save && npm explore @daostack/infra -- npm install openzeppelin-solidity --no-save && npm explore @daostack/arc-hive -- npm install openzeppelin-solidity --no-save",
+    "verify.build": "node node_modules/typescript/bin/tsc verify.ts -sourceMap",
+    "verify.help": "node verify.js -h",
+    "verify.lint": "tslint -c tslint.json verify.ts",
```

Comment:
Could be more than just about `verify.ts` (all typescript files in the project)

Reply:
sorry, I don't understand the suggestion.  are you suggesting to convert all scripts to ts?

Reply:
lint everything

Reply:
for now, verify is all there is.  we can cross the wider bridge when we come to it.